client/src/contexts/Okteto.context.tsx

```diff
@@ -92,6 +93,17 @@ const OktetoProvider = ({ children } : OktetoProviderProps) => {
     }
   };
 
+  // relaunch the current environment
```

Comment:
The function name is self-explanatory, I would remove this comment. Or we could rename it to `relaunchCurrentEnvironment`

```suggestion
```